module/VuFind/src/VuFind/Search/SearchOrigin/AbstractSearchOrigin.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Search Origin Object
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
```

Comment:
I notice a few copied-and-pasted 2010 copyright statements. You might want to change these to 2024, and you're welcome to change the institution to your own if you wish.